go.mod

```
@@ -0,0 +1,11 @@
+module main
+
+go 1.18
```

```suggestion
go 1.18
```
I am using the latest one and not sure if it will compatible with your local setup 1.xx or bring any other conflicts to run tests. 🤞